src/Scaffolding/VS.Web.CG.Mvc/BlazorIdentity/BlazorIdentityGenerator.cs

```diff
@@ -80,7 +80,7 @@ private IEnumerable<string> GeneralT4Files
         {
             get
             {
-                if (_generalT4Files == null || !_generalT4Files.Any())
+                if (_generalT4Files == null)
```

Comment:
Should have commented on this earlier, but its more obvious now. The back and forth of `== null` sometimes and `is null` other times makes my OCD hurt :) Can we standardize on `is null`?